CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Enhancements
+* [artifact-manager] Issue 220 / mvn archetype:generate for abx project - updated the package.json template for generating abx actions
```

Comment:
Thanks for updating this! Can you also add an update to the RELEASE.md file

Reply:
Done